GeneratorInterface/Pythia8Interface/src/ResonanceDecayFilterHook.cc

```diff
@@ -28,7 +30,13 @@ bool ResonanceDecayFilterHook::initAfterBeams() {
     if ( (did == 14 || did == 16) && allNuAsEquivalent_) {
       did = 12;
     }
-    
+    if ( (did == 2 || did==3 || did==4 || did==5 ) && udscAsEquivalent_) {
+      did = 1;
+    }
+    if ( (did == 2 || did==3 || did==4 || did==5 || did==6) && udscbAsEquivalent_) {
+      did = 1;
+    }
+
```

Comment:
Yes, of course :) sorry about that. 

I'll just make a new pull request and this one can be killed.